tests/integration_tests/performance/configs/test_network_latency_config_5.10.json

```diff
@@ -211,12 +211,7 @@
     },
     "measurements": {
         "latency": {
-            "statistics": [
-                {
-                    "criteria": "EqualWith",
-                    "function": "Avg"
-                }
-            ],
+            "statistics": [],
```

Comment:
I guess we probably want to do this for all host kernels, not just 5.10?

Reply:
Makes sense, updated.